website/docs/guides/using-with-composedb-client.md

```diff
@@ -0,0 +1,79 @@
+# Using With ComposeDB Client
+
+[ComposeDB](https://composedb.js.org) is a set of TypeScript libraries and tools to interact with the [Dataverse](https://blog.ceramic.network/into-the-dataverse/) using the [Ceramic network](https://ceramic.network/).
+
+First, you should start with creating your instance of `ComposeClient` from `@composedb/client` package, passing it the
+url to the ceramic node you want to use and the runtime composite definition of the composite you want to use in your App.
+
+```ts
+import { ComposeClient } from '@composedb/client'
+import { definition } from './__generated__/definition.js'
+
+const compose = new ComposeClient({ ceramic: 'http://localhost:7007', definition })
+```
+
+Next, you can create a DID Session, passing it the resources from your client instance. The resources are a list of model
+stream IDs from your runtime composite definition:
+
+```ts
+import { DIDSession } from 'did-session'
+import { EthereumAuthProvider } from '@ceramicnetwork/blockchain-utils-linking'
+
+const ethProvider = // import/get your web3 eth provider
+const addresses = await ethProvider.enable()
+const authProvider = new EthereumAuthProvider(ethProvider, addresses[0])
+
+const loadSession = async(authProvider: EthereumAuthProvider, resources: Array<string>):Promise<DIDSession> => {
+  return await DIDSession.authorize(authProvider, { resources: resources})
+}
+
+const session = await loadSession(authProvider, compose.resources)
+```
+
+Next, you can assign the authorized did from your session to your client. 
+
+```ts
+compose.setDID(session.did)
+
+// use the compose instance to make queries in ComposeDB graph
+```
+
+Before you start making mutations with the client instance, you should make sure that the session is not expired
+```ts
+// before compose mutations, check if session is still valid, if expired, create new
+if (session.isExpired) {
+  const session = loadSession(authProvider)
+  compose.setDID(session.did)
+}
+
+// continue to make mutations
+```
```

Comment:
maybe move to manage session section, and would maybe structure closer to original, managing a session is pretty independent of compose/httpclient/glaze and even ceramic 

mostly in composedb section would just want to emphasize using compose.reources for authorization, most other details are independent of composedb

Reply:
I was thinking about having this guide as a place where anybody who wants to use did-session with ComposeDB can come to, copy/paste the code and get going, with a minimal understanding of why each piece of the code is useful.

So in my mind, this guide doesn't have to have only information specific to ComposeDB, but rather all the sufficient information to get going.

Conversely, I wanted to avoid any ComposeDB-specific information outside of the guides related directly to ComposeDB, because I thought that this would confuse people when it comes to understanding whether did-session is independent or not from ComposeDB.

Does that make sense?

In the same time, I think it's a good call that I should copy the part about checking, if session has expired to the manage section :).